inline-autofix.py

```diff
@@ -0,0 +1,4 @@
+2 == 2
```

Comment:
This expression is always True: `2 == 2` or `2 != 2`. If testing for floating point NaN, use `math.isnan(2)`, or `cmath.isnan(2)` if the number is complex.
#
<a href="https://dev.semgrep.dev/api/agent/deployment/37/issues/11309/sentiment?hash=cef1ff82c64a53805d683a13cc2119cfa462a84dd412673d32b5b6ed10fe1959&sentiment=bad?utm_campaign=finding_notification&utm_medium=review_comment&utm_source=github&utm_content=logo">
<img align="right" src="https://img.shields.io/badge/-Not%20helpful%20%F0%9F%91%8E-%235F36D9" height=24px alt="" />
</a>
<sub>🔴 Fix or <a href='https://semgrep.dev/docs/ignoring-files-folders-code/#reference-summary'>ignore</a> this finding to merge your pull request.</sub><br/>
<sub>🙈 From <a href='https://semgrep.dev/r/python.lang.correctness.useless-eqeq.useless-eqeq?utm_campaign=finding_notification&utm_medium=review_comment&utm_source=github&utm_content=rule'>python.lang.correctness.useless-eqeq.useless-eqeq</a>.</sub>
<!---

🤫 WELCOME TO SECRET SEMGREP! 🤫
This information is for debugging purposes and does not appear in rendered PR comments.

Finding id: 11309
Syntactic id: 4dae33596d66438f61304ec12fdb4b3f
Start line: 1,1
End line: 1,7
Index: 0
-->